### DISCUSSION TITLE
Add to Curseforge?

### DISCUSSION THREAD

**Discussion by linfoxgamer** [anatom3000/GuessWhatWillHappenInThis#49] [Ideas]
This is a Good Mod,A Good Idea!

But is very Good adding to curseforge,to make it More Popular and more People and Developers will Discover This

Curseforge is a Very Popular site to Upload Mods
https://www.curseforge.com/minecraft/mc-mods?filter-game-version=&filter-sort=2

**Comment by zowepsilon** [comment]
thanks, i know curseforge but sadly it got rejected so no curseforge, sorry :/

**Comment by zowepsilon** [comment]
![image](https://user-images.githubusercontent.com/49984698/120328280-f7c5c000-c2ea-11eb-8981-18944b4c3534.png)
![image1](https://cdn.discordapp.com/attachments/836125047781326862/844985377768341534/unknown.png)

**Comment by zowepsilon** [comment]
also if you want to => https://discord.gg/VenFqgaHDj

**Comment by linfoxgamer** [comment]


**Reply by linfoxgamer** [reply]
I Think is Rejected because "All Sounds are Blyat" lol
or because the F#ck in Description of Source Code

I Think Is Not Owofy Messages Because Another Mods Have Owofy
https://www.curseforge.com/minecraft/mc-mods/hewo
https://www.curseforge.com/minecraft/mc-mods/l12n
https://www.curseforge.com/minecraft/mc-mods/owolib

**Reply by zowepsilon** [reply]
it has been rejected anyway, so...
